### PR TITLE
#1016 - Fix

### PR DESCRIPTION
**Pull request by Mihir-solanki-13**
We created a total of 4 new endpoints - 3 of which are the main form pages and 1 is  create work , these end points were crashing and not connected to  the respective  urls ,
Dashboard_create_project
Dashboard_view_project

### CODE REVIEW

File: FusionIIIT/templates/iwdModuleV2/viewWork.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="fields">
     <div class="eight wide field">
-        <form class="ui form" style="padding: 8px; padding-left: 24px; padding-right: 24px;", method="post" action="http://172.27.16.216/iwdModuleV2/page1View/">{% csrf_token %}
+        <form class="ui form" style="padding: 8px; padding-left: 24px; padding-right: 24px;", method="post" action="page1View">{% csrf_token %}
```
Comment (divij1822) [inline FusionIIIT/templates/iwdModuleV2/viewWork.html:3]: the page is not being redirected to page1View

**Review by divij1822 — COMMENTED** [review]
I have added a comment in the code. PTAL and address the changes.

File: FusionIIIT/templates/iwdModuleV2/viewWork.html (hunk 1 above)
Comment (Mihir-solanki-13) [reply FusionIIIT/templates/iwdModuleV2/viewWork.html:3]: Here our task was just to make end point and as create view requisition was not showing anything I do this.
For proper redirect to page1View that was this weeks task I have generated a PR Today
https://github.com/FusionIIIT/Fusion/pull/1073

**Review by divij1822 — COMMENTED** [review]
Approved. Ping @akshatnema

**Comment by akshatnema** [comment]
Closing on the initiation of #1092

**Pull request closed without merging**